integration_test/third_party_apps_test/applications/dcgmv1/metadata.yaml

```diff
@@ -0,0 +1,161 @@
+gpu_platforms: # p4, p100 don't emit DCGM profiling metrics
+  - model: a100
+    platforms:
+      - ubuntu-os-cloud:ubuntu-2004-lts
+  - model: v100
+    platforms:
+      - ubuntu-os-cloud:ubuntu-2004-lts
+  - model: t4
+    platforms:
+      - ubuntu-os-cloud:ubuntu-2004-lts
+  - model: l4
+    platforms:
+      - debian-cloud:debian-11
+      - ml-images:common-gpu-debian-11-py310
+      - rocky-linux-cloud:rocky-linux-8
+      - rocky-linux-cloud:rocky-linux-9
+      - suse-cloud:sles-15
+      - ubuntu-os-cloud:ubuntu-2004-lts
+      - ubuntu-os-cloud:ubuntu-2204-lts
+expected_metrics:
+  - type: workload.googleapis.com/dcgm.gpu.profiling.sm_utilization
+    value_type: DOUBLE
+    kind: GAUGE
+    description: Fraction of time at least one warp was active on a multiprocessor, averaged over all multiprocessors.
+    monitored_resources: [gce_instance]
+    labels:
+      - &model_label
+        name: model
+        value_regex: .*
+        description: GPU model name.
+      - &uuid_label
+        name: uuid
+        value_regex: .*
+        description: GPU universally unique identifier.
+      - &gpu_number_label
+        name: gpu_number
+        value_regex: "[0-9]*"
+        description: GPU index starting at 0.
+      - &instrumentation_source_label
+        name: instrumentation_source
+        value_regex: agent.googleapis.com/dcgm
+    notes:
+      - &dcgm_profiling_note
+        Not available on GPU models P100 and P4.
+    representative: true
+  - type: workload.googleapis.com/dcgm.gpu.profiling.sm_occupancy
+    value_type: DOUBLE
+    kind: GAUGE
+    description: Fraction of the number of warps resident on a multiprocessor, averaged over all multiprocessors.
+    monitored_resources: [gce_instance]
+    labels:
+      - *model_label
+      - *uuid_label
+      - *gpu_number_label
+      - *instrumentation_source_label
+    notes:
+      - *dcgm_profiling_note
+  - type: workload.googleapis.com/dcgm.gpu.profiling.pipe_utilization
+    value_type: DOUBLE
+    kind: GAUGE
+    description: Fraction of cycles the corresponding GPU pipe was active, averaged over time and all multiprocessors.
+    monitored_resources: [gce_instance]
+    labels:
+      - *model_label
+      - *uuid_label
+      - *gpu_number_label
+      - name: pipe
+        value_regex: tensor|fp64|fp32|fp16
+        description: "GPU pipe in use, one of [tensor, fp64, fp32, fp16]."
+        notes:
+          - For L4, the `pipe` value `fp64` is not supported.
+      - *instrumentation_source_label
+    notes:
+      - *dcgm_profiling_note
+  - type: workload.googleapis.com/dcgm.gpu.profiling.dram_utilization
+    value_type: DOUBLE
+    kind: GAUGE
+    description: Fraction of cycles data was being sent or received from GPU memory.
+    monitored_resources: [gce_instance]
+    labels:
+      - *model_label
+      - *uuid_label
+      - *gpu_number_label
+      - *instrumentation_source_label
+    notes:
+      - *dcgm_profiling_note
```

Comment:
```suggestion
```
`dram_utilization` (bandwidth_utilization) now has fallback so we can remove the note.

Reply:
So it does. I forgot that v1 and v2 reuse the same underlying implementation. Thanks for catching this.